<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Various Machine Learning Techniques for OCR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>501</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBoxController">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>242</height>
        </size>
       </property>
       <property name="title">
        <string>Menu</string>
       </property>
       <widget class="QPushButton" name="pushButtonLoadFile">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>79</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Load images you want to recognize characters in</string>
        </property>
        <property name="text">
         <string>Load File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCamera">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>79</width>
          <height>24</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>Load images you want to recognize characters in</string>
        </property>
        <property name="text">
         <string>Camera</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>211</width>
          <height>58</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="labelAlgorithmChoice">
           <property name="text">
            <string>Choose an algorithm</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxAlgorithms"/>
         </item>
        </layout>
       </widget>
       <widget class="QSlider" name="horizontalSliderMaxH">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderMinH">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>400</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderMaxW">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderMinW">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>220</y>
          <width>151</width>
          <height>18</height>
         </rect>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxH">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>140</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>maxH</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinH">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>minH</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxW">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>200</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>maxW</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinW">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>220</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>minW</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBoxImageAfterOCR">
       <property name="minimumSize">
        <size>
         <width>247</width>
         <height>241</height>
        </size>
       </property>
       <property name="title">
        <string>Image after recognition</string>
       </property>
       <widget class="QLabel" name="labelImageAfterOCR">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>220</width>
          <height>200</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBoxTextOCR">
       <property name="minimumSize">
        <size>
         <width>246</width>
         <height>241</height>
        </size>
       </property>
       <property name="title">
        <string>Recognized Text</string>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>221</width>
          <height>201</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxLoadImage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>247</width>
         <height>242</height>
        </size>
       </property>
       <property name="title">
        <string>Loaded Image</string>
       </property>
       <widget class="QLabel" name="labelImage">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>220</width>
          <height>200</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
